site/assignments/zombies/zombies.rst

```diff
@@ -210,58 +221,58 @@ General FAQ:
    * I know... it should... it's still stochastic...
 
 * Does my text file have enough details?
-	* Probably. The shorter the better. I really just want to see that you played around a little.
+    * Probably. The shorter the better. I really just want to see that you played around a little.
```

Comment:
These need revisiting.

Reply:
Do you mean 
(a) the FAQ in general
(b) this one needs to be changed because it's straight up about A1
(c) I had thought of a 3rd potential reason before, but whatever, I forget.

Reply:
The faq in general. The answers are far too informal, imo.

Reply:
I gotta' re-work them in general across the assignments. I liked the idea of a common page so it wasnt' copy/paste, but some assignments have specifics. Either way, future problem.

Reply:
Yeah, not blocking this, hence the ship it.